content/_releases/22/34.md

```diff
@@ -0,0 +1,171 @@
+    *   `attributes.docker_registry.uri` - Docker Registry URI
+    *   `attributes.docker_registry.credentials` - Docker Registry credentials
+*   `docker_repo_name` and `docker_target_tag` in the `POST` HTTP call to `/v2/components` (can be set if its team has `docker_registry` object)
+    *   `attributes.docker_repo_name` - String representing component's name in docker registry. Available only if team supports docker registry
+    *   `attributes.docker_target_tag` - String representing component's tag in docker registry. Available only if team supports docker registry
+
+> Please note, as of this stage these attributes have no functionality attached to them.
+
+{% if site.data.tenant.name == "elastic.io" %}
+
+## HELM3
+
+We constantly improve different aspects of HELM3 deployments in collaboration
+with our partners. This section lists updates and improvements included in this release.
+
+### Dashboard Runlog retention period
+
+The flow execution statistics presented on the Dashboard Runlog is stored
+in the platform Mongo Database. This is in contrast to the flow execution threads, shown
+on the Executions pages, which is stored separately in the Clickhouse.
+
+When you access the Dashboard Runlog, the platform queries the MongoDB to get these
+records. This call to the Database can slow-down the page load for the high-end users.
+For this purposes it is prudent to limit the retention period of data in the Database.
+
+Previously you could only control the retention period by enforcing `expireAfterSeconds`
+option in the Database index holding the execution records. In this release we
+introduce a new `TASK_STAT_START_INDEX_TTL` environment variable to the HELM3 charts,
+which you can use to set retention suitable for your environment. If not specified,
+the **default value is set 432000 seconds (5 days)**. Which means the platform will store
+the Dashboard Runlog record only for 5 days in the Database.
+
+#### Breaking Changes
+
+Setting the `TASK_STAT_START_INDEX_TTL` environment variable to control the retention
+period of the Dashboard Runlog is recommended for the new installations of the platform.
+
+If your platform installation already has the retention period set in the Database
+prior to your installation of the platform {{ page.version }} version then your
+environment variable [will not be applied](https://www.mongodb.com/docs/v4.2/reference/command/createIndexes/#index-options). If you need to change the value you must
+change the Database index setup prior to the deployment of the current release.
+
+If you need to change the retention period of Dashboard Runlog in the future, remove
+the `expireAfterSeconds` option of the Database index holding the Dashboard Runlog records,
+set the value `TASK_STAT_START_INDEX_TTL` of environment variable and then deploy
+the platform version again.
+
+### Kubernetes 1.22
+
+Our platform is now compatible with the Kubernetes version `1.22`.
```

Comment:
Check out the new features enabled here
https://kubernetes.io/blog/2021/08/04/kubernetes-1-22-release-announcement/